README.md

```diff
@@ -4,19 +4,56 @@ The project will create the next breathtaking breakthrough in this planet's hist
 # How to build
 On a normal day, open your *nix terminal and hope for the best
 
+_You may wish to choose an appropriate generator option to match your environment_
```

Comment:
It appears to not need a default on my MacBook.  I'm not sure if Makefiles will work on Windows, but would rather use VS.  But there are so many flavors of VS.  Is there a mechanism to prefer Makefiles on *nix boxes, and to choose a specific version of VS on Windows boxes?  Or, if the defaults generator Does the Right Thing(tm), let's let it keep doing that.  No sense in fighting a system that's trying to help us.